pkg/hs/urbit-king/lib/Urbit/Vere/Pier.hs

```diff
@@ -183,7 +183,7 @@ bootNewShip pill lite ship bootEv = do
   let logPath = (pierPath </> ".urb/log")
 
   rwith (Log.new logPath ident) $ \log -> do
-    logInfo "Event log onitialized."
+    logInfo "Event log initialized."
```

Comment:
Are you sure? "Onitialized" here means that's it's been turned on and initialized; you lose a shade of meaning.

Reply:
Reverted.

Reply:
wat. I was not being serious

Reply:
Please do not do that in a code review.